source/templates.tex

```diff
@@ -7145,9 +7148,9 @@
 as part of the substitution
 process.
 \end{note}
-Only invalid types and expressions in the immediate context of
-the function type,
-its template parameter types,
+Invalid types and expressions only in the immediate context of
+the function's type outside of the exception specification,
+its template parameters,
```

Comment:
@jensmaurer  Could we change `template parameters` to *template-parameters*; It would indicate the deduction failures in default template arguments are also covered by these rules.